app/config/databases.php

```diff
@@ -10,6 +10,10 @@
         'username' => '',
         'password' => '',
     ],
+    'sandbox' => [
```

Comment:
Change to default, to use the convention in new AR.

Reply:
Done.